earth_enterprise/rpms/opengee-common/post-uninstall.sh

```diff
@@ -22,18 +22,18 @@ BININSTALLROOTDIR="/etc/init.d"
 
 #------------------------------------------------------------------------------
 # Group names:
-GEGROUP_NAME=$(cat "$BININSTALLROOTDIR/gevars.sh" | grep GEGROUP | cut  -d'=' -f2)
-
 
 remove_users_groups()
 {
-    if [ -n "$(getent group "$GEGROUP_NAME")" ]; then
-        groupdel "$GEGROUP_NAME"
-    fi
+    echo "opengee group $GEGROUP may now be safely removed."
```

Comment:
Style: 'opengee' -> 'OpenGEE'

Minor: The group can be safely removed, only if the system administrator can confirm there are no more files or directories owned by it.